README.md

```diff
@@ -138,14 +145,30 @@ Run `ng generate component component-name` to generate a new component. You can
 
 Run `npm run build` to build the project. The build artifacts will be stored in the `dist/` directory.
 
-### Running unit tests
+To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page. This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 16.2.7; it was then updated to Angular 17.
 
-Run `npm run test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+### Docker deployment
 
-### Running end-to-end tests
+A Dockerfile is provided in order to serve the webapp through a container.
 
-Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To use this command, you need to first add a package that implements end-to-end testing capabilities.
+Build the webapp with the following command:
 
-### Further help
+```bash
+# Default build / test the image locally
+ng build --configuration=production
 
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page. This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 16.2.7; it was then updated to Angular 17.
+# Serve the app from the /myapp prefix if you use a k8s cluster that shares the same domain for multiple services.
+ng build --configuration=production --base-href /myapp/
```

Comment:
Presumably you can run it locally with `ng serve` still also? I think that instruction is probably helpful for fast iteration on the UI (e.g. it has live reload); I think docker based building and serving is useful for easy generic deployment; but probably not needed fore firebase... ?

Reply:
Yes, these actually are the production deployment instructions, I forgot to mention the usual `ng serve` / `npm run start` for development...

Reply:
clarified in 14e7118ccb970d41a3fd2a439e2538d472e289a4